index.html

```diff
@@ -1505,12 +1505,12 @@ <h3>Algorithm</h3>
           <a data-link-for="JsonLdErrorCode">keyword redefinition</a> error has
           been detected and processing is aborted.</li>
         <li><span class="changed">Otherwise</span>, since <a>keywords</a> cannot be overridden,
-          <var>term</var> MUST NOT be a <a>keyword</a> and a
-          <a data-link-for="JsonLdErrorCode">keyword redefinition</a>
-          error has been detected and processing is aborted.
-          If <var>term</var> has the form of a keyword
+          <var>term</var> MUST NOT be a <a>keyword</a>,
```

Comment:
Actually "MUST NOT have the form of a keyword" would be enough,
since all actual keywords also have the form of a keyword.
But it might be better to be explicit about the fact that we reject both actual keywords and would-be keywords.

Reply:
That was my thinking. An implementation can always choose to do this optimization.